### DISCUSSION TITLE
can we change the .alive message?

### DISCUSSION THREAD

**Discussion by Pasindu208** [phaticusthiccy/WhatsAsenaDuplicated#133] [Q&A] [answered]
Is it possible to change the reply for the **.alive** command?
If so please tell me how.

**Comment by phaticusthiccy** [comment, marked as answer]
Use `.degis` // `.setvar ALIVE_MESSAGE:msg` or try on heroku config vars. ( Heroku >> app >> Settings >> Reavel Config Vars )

**Reply by Pasindu208** [reply]
Thank you!
will i be able to include an image?

**Reply by phaticusthiccy** [reply]
Check out .degis 
You will see variables
